models/issue.go

```diff
@@ -1600,28 +1604,59 @@ func SearchIssueIDsByKeyword(kw string, repoIDs []int64, limit, start int) (int6
 }
 
 // UpdateIssueByAPI updates all allowed fields of given issue.
-func UpdateIssueByAPI(issue *Issue) error {
+func UpdateIssueByAPI(issue *Issue, doer *User) (*Comment, bool, error) {
```

Comment:
can you add to the func desc what it will returns since the return var are not specified

Reply:
done